website/homepage/src/docs/obj_doc_template.html

```diff
@@ -110,7 +110,7 @@ <h4 class="mt-4 p-3 font-semibold">Example Usage</h4>
 
     {% elif obj["demos"] %}
 
-    <div class="container relative">
+    {# <div class="container relative">
```

Comment:
Why did you comment this out?

Reply:
oh I just find it a waste of space. will uncomment, we can redesign later